lib/replicache.dart

```diff
@@ -342,6 +347,23 @@ class Replicache implements ReadTransaction {
     if (onChange != null) {
       scheduleMicrotask(onChange);
     }
+
+    // TODO(arv): We should pass the ref of the current commit to ensure we get
+    // consistent results in all the subscribe callbacks.
+    _changeStreamController.add(null);
```

Comment:
Initially I wrapped this in a transaction and yielded the tx in the internal stream. I thought I could share the tx between all the subscribe callbacks. Turns out that streams are done in microtasks so the tx got closed before the callbacks were invoked.

Reply:
I have a followup. I'm going to land this to get another PR with those changes.